core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -410,6 +416,14 @@ public String getHistoryUUID() {
     }
 
     /**
+     * Returns how many bytes we are currently moving from heap to disk
```

Comment:
nit: shall we be more concrete and say how much bytes are moving from the indexing buffer to segments?